tests/welcome.test.js

```diff
@@ -0,0 +1,21 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../src/index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Welcome route', () => {
+    it('Welcome route should return object', (done) => {
+        chai
+        .request(server)
+        .get('/')
+        .end((err, res) => {
+            res.should.have.status(200);
+            res.body.should.be.a('object');
+            res.body.message.should.be.a('string');

```

Comment:
Can you change the res.body.message.should.be.a('string') to something like expect(res.body.message).to.equal('The actual message'); if you have any difficulties doing this I can help.

Reply:
> Can you change the res.body.message.should.be.a('string') to something like expect(res.body.message).to.equal('The actual message'); if you have any difficulties doing this I can help.

i corrected it, by the way thank you @Muhire-Josue